test/MyFirstAPIConsume.test.js

```diff
@@ -0,0 +1,62 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+describe('First Api Tests', () => {
+    it('Consume GET Service', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+            name: 'John',
+            age: '31',
+            city: 'New York'
+        };
+
+        const response = await axios.get('https://httpbin.org/get', { query });
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+
+    it('Consume HEAD Service', async () => {
+        const response = await axios.head('https://httpbin.org/');
+        expect(response.status).to.equal(StatusCodes.OK);
+    });
+
+    it('Consume PUT Service', async () => {
+        const dataput = {
+            name : 'Eduardo',
+            age : '28',
+            city : 'Mocoa'
+        }
+        const response = await axios.put('https://httpbin.org/put',{ dataput });
```

Comment:
```suggestion
        const response = await axios.put('https://httpbin.org/put', dataput );
```
For put, the parameter received is a simple json, which is already the structure of the dataput variable. (For get it is different because it is a query, not a body)